### PR TITLE
[MIG] product_packaging_type_pallet: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Dependencies:
- product_packaging_dimension: https://github.com/OCA/product-attribute/pull/799
- product_packaging_type: https://github.com/OCA/product-attribute/pull/760

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]
I approve the migration through basically it only consists in changing the version number to 14, so I'm not too sure if it is fair to make you a listed module contributor (this would not scale). I guess this is to be decided with CampToCamp authors. cc @jgrandguillaume

**Review by dsolanki-initos — APPROVED** [review]
Code and functional test . LGTM

**Comment by dsolanki-initos** [comment]
It seems that this PR is ready to merge. It would be great if someone could merge this PR.

**Comment by dsolanki-initos** [comment]
Can somone please merge this PR?

**Comment by dsolanki-initos** [comment]
@OCA/product-maintainers  can this be merged?

**Comment by dsolanki-initos** [comment]
It would be great if somone could merge this PR.

**Comment by dsolanki-initos** [comment]
It would be great if we could merge this PR.

**Comment by dsolanki-initos** [comment]
It would be great if we could merge this PR.

**Comment by dsolanki-initos** [comment]
IT would be great if the maintainers could merge it since long it is ready.

**Comment by dsolanki-initos** [comment]
It would be great if the maintainers could merge it since long it is ready.

**Comment by dsolanki-initos** [comment]
It seems that PR is ready to merge . It would be great if maintainer merge it.

**Comment by dsolanki-initos** [comment]
It seems that PR is ready to merge . It would be great if maintainer merge it.

**Comment by dsolanki-initos** [comment]
Hi @rvalyi 
It seems that your Merge command is aborted. Will you please try again?
Thank You !

**Comment by rousseldenis** [comment]
@rvalyi @dzungtran89 #799 is not merged yet. This one should not be merged before.

**Comment by rousseldenis** [comment]
> @rvalyi @dzungtran89 #799 is not merged yet. This one should not be merged before.

Ok, another PR was done.

/ocabot merge patch

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo14-addon-product-packaging-type @ git+https://github.com/trobz/product-attribute@14.0-mig-product_packaging_type#subdirectory=setup/product_packaging_type
+odoo14-addon-product-packaging-dimension @ git+https://github.com/dzungtran89/product-attribute@14.0-mig-product_packaging_dimension#subdirectory=setup/product_packaging_dimension
```
Comment (rousseldenis) [inline test-requirements.txt]: @dzungtran89 Could you remove this as already merged ?

Comment (dzungtran89) [reply test-requirements.txt]: sure, I updated it, including a fix regarding to a new change on https://github.com/OCA/product-attribute/pull/850

**Pull request merged**